com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicSubD.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine.Formats.Alembic.Sdk;
+
+namespace UnityEngine.Formats.Alembic.Importer
+{
+    internal class AlembicSubD : AlembicMesh
+    {
+        aiSubD m_abcSchema;
+
+        internal override aiSchema abcSchema
+        {
+            get { return m_abcSchema; }
+        }
+
+        internal override void AbcSetup(aiObject abcObj, aiSchema abcSchema)
+        {
+            base.AbcSetup(abcObj, abcSchema);
```

Comment:
Do think this is needed ?

Larger picture: Is there any value in having a separate AlembicNode type in C# ? Could we have both polyMesh and subD c++ classes always create the same C# PolyMesh type ?
SubD is nothing more than a mesh today

Reply:
That's a good point.  I thought about that, and didn't know if it was worth having something as a placeholder for future attribute possibilities (crease weights, etc.).  But, might just be confusing since it doesn't do anything more than a Mesh.

Reply:
You might be right.
Let's leave it as is for now: Maybe the adaptive GPU Subdividing project will catch on and there'll be interest for creasing info.